components/FlexibleHeader/tests/snapshot/FlexibleHeaderViewAnimationDelegateTests.swift

```diff
@@ -83,15 +103,56 @@ class FlexibleHeaderViewAnimationDelegateTests: XCTestCase {
     window.layer.speed = 100000
     CATransaction.flush()
 
-    let mockDelegate = MockMDCFlexibleHeaderViewAnimationDelegate(didCompleteExpectation:
-        expectation(description: "didComplete"))
+    let didCompleteExpectation = expectation(description: "didComplete")
+    let mockDelegate =
+      MockMDCFlexibleHeaderViewAnimationDelegate(didChangeAnimatedExpectation: nil,
+                                                 didChangeNotAnimatedExpectation: nil,
+                                                 didCompleteExpectation: didCompleteExpectation)
     fhv.animationDelegate = mockDelegate
 
     // When
     fhv.trackingScrollView = scrollView1
     fhv.trackingScrollView = scrollView2
 
     // Then
-    wait(for: [mockDelegate.didCompleteExpectation], timeout: 0.1)
+    wait(for: [didCompleteExpectation], timeout: 0.1)
+  }
+
+  func testAnimationCallbackIsInvokedNotAnimatedWhenTheTrackingScrollViewIsChanged() {
```

Comment:
I'm a little confused by the name of this test vs. its behavior. "AnimationCallbackIsInvokedNotAnimated" makes me expect that the `didChangeNotAnimatedExpectation` expectation will be fulfilled, but the test checks for the `didChangeAnimatedExpectation`. Am I misreading this?

Reply:
I updated the title to `testAnimationCallbackIsInvokedWithAnimatedYesWhenTheTrackingScrollViewIsChanged`.